src/components/Table/TableColumnHeader.tsx

```diff
@@ -93,12 +92,20 @@ export const TableColumnHeader = ({
     return (
         <th
             ref={ref}
-            className="tw-text-xs tw-font-medium tw-text-black-100 dark:tw-text-white tw-px-4 tw-py-3 tw-outline-none tw-cursor-pointer tw-group focus-visible:bg-violet-90"
+            className={`tw-text-xs tw-font-medium tw-text-black-100 dark:tw-text-white tw-px-4 tw-py-3 tw-outline-none tw-group focus-visible:bg-violet-90 ${
```

Comment:
Just for the sake of consistency, shouldn't we use the `merge` utility here?

Reply:
Yes we should 👌